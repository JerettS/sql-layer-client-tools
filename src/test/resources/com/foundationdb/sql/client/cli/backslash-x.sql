\x
SELECT 1,2,3,4;
\x
SELECT 1,2,3,4;
\x
SELECT 1 foo, 2 a;

CREATE TABLE t ( label int, long_label int, a int);
INSERT INTO t (label, long_label, a) values (1,2,3), (4,5,6), (7,8,9);
SELECT * FROM t;

\x on
\x ON
\x off
\x OFF
\x hi
