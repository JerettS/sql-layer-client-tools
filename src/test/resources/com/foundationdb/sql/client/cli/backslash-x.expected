Expanded display is on
-[ RECORD 1 ]
_SQL_COL_1 | 1
_SQL_COL_2 | 2
_SQL_COL_3 | 3
_SQL_COL_4 | 4

Expanded display is off
 _SQL_COL_1 | _SQL_COL_2 | _SQL_COL_3 | _SQL_COL_4 
------------+------------+------------+------------
          1 |          2 |          3 |          4 
(1 row)

Expanded display is on
-[ RECORD 1 ]
foo | 1
a   | 2

ROWS: 0

ROWS: 3

-[ RECORD 1 ]
label      | 1
long_label | 2
a          | 3
-[ RECORD 2 ]
label      | 4
long_label | 5
a          | 6
-[ RECORD 3 ]
label      | 7
long_label | 8
a          | 9

Expanded display is on
Expanded display is on
Expanded display is off
Expanded display is off
Wrong argument type: expected [on|off]